op_crates/web/08_text_encoding.js

```diff
@@ -124,13 +124,15 @@
   }
 
   function btoa(s) {
+    s = String(s);
     const byteArray = [];
     for (let i = 0; i < s.length; i++) {
       const charCode = s[i].charCodeAt(0);
       if (charCode > 0xff) {
-        throw new TypeError(
+        throw new DOMException(
           "The string to be encoded contains characters " +
             "outside of the Latin1 range.",
```

Comment:
Unrelated, but why is this a string concat?

Reply:
An attempt at formatting was made?
Collapsed them.